front/app/components/Form/Components/ErrorDisplay.tsx

```diff
@@ -57,7 +57,7 @@ const ContainerInner = styled.div`
   justify-content: space-between;
   padding: 10px 13px;
   border-radius: ${(props) => props.theme.borderRadius};
-  background: ${colors.red100};
+  background: ${colors.errorLight};
```

Comment:
TBH I am not a super big fan of these semantic color names. I prefer the `red100` here. `errorLight` doesn't tell me anything, except that it's an 'error color'- which I already knew because it's used on an `ErrorDisplay` component. At least `red100` tells me it's a light red color. I can understand keeping some of them around for legacy reasons but I think we should switch the design system names (so `red100` etc) as much as possible. WDYT?

Reply:
I guess that's also what you've already done in most places! Anyway just wanted to mention it, would prefer to move away from those semantic colors at some point

Reply:
Hmm ok yeah that makes sense!